ksp_plugin_adapter/ksp_plugin_adapter.csproj

```diff
@@ -57,16 +57,36 @@
   </PropertyGroup>
   <ItemGroup Condition="'$(Configuration)' == 'Release' Or '$(Configuration)' == 'Debug'">
     <Reference Include="Assembly-CSharp">
-      <HintPath>..\..\KSP Assemblies\1.4.0\Assembly-CSharp.dll</HintPath>
+      <HintPath>..\..\KSP Assemblies\1.4.1\Assembly-CSharp.dll</HintPath>
       <Private>False</Private>
     </Reference>
     <Reference Include="System" />
     <Reference Include="UnityEngine">
-      <HintPath>..\..\KSP Assemblies\1.4.0\UnityEngine.dll</HintPath>
+      <HintPath>..\..\KSP Assemblies\1.4.1\UnityEngine.dll</HintPath>
+      <Private>False</Private>
+    </Reference>
+    <Reference Include="UnityEngine.CoreModule">
+      <HintPath>..\..\KSP Assemblies\1.4.1\UnityEngine.CoreModule.dll</HintPath>
```

Comment:
So as expected these references are generating warnings which cause the continuous build to fail.  There does not seem to be any way to selectively ignore linker warnings in MSBuild.

Ideally, we would want these references to be only active on Mac (and possibly Linux).  According to [this page](http://www.mono-project.com/archived/porting_msbuild_projects_to_xbuild/) it is possible to have references that are conditioned by the operating system.  Could you try this?

Reply:
FYI, on Windows, `$(OS)` is `Windows_NT`.

Reply:
Hopefully fixed now